packages/core/strapi/lib/core/bootstrap.js

```diff
@@ -22,4 +23,13 @@ module.exports = function({ strapi }) {
   if (!shouldServeAdmin) {
     strapi.config.serveAdminPanel = false;
   }
+
+  // ensure public repository exists
+  try {
+    await fse.ensureDir(strapi.dirs.public);
+  } catch (e) {
+    throw new Error(
```

Comment:
I think that won't be enough. If you have a read/write error you won't know and we will display the wrong message. I recommend you use pathExists to specificaly verify that info.
Should we only use ensureDir in dev mode by the way?

Reply:
You're right, I can just use pathExists. I would say even in dev mode so the user knows they will need to do the same with deploying Strapi. It's also reassuring we don't create folders/files oustide of the Strapi project